packages/protocol/src/mdns/MdnsServer.ts

```diff
@@ -252,7 +252,7 @@ export class MdnsServer {
     }
 
     async announce(announcedNetPort?: number) {
-        await Promise.all(
+        await Promise.allSettled(
```

Comment:
This will hide errors, should it throw an aggregate error if there are failures?  Even if there shouldn't be errors I don't like the idea of them disappearing

Reply:
Maybe we should have a utility so we can do e.g. `await MatterAggregateError.throwSettlementErrors(...)` or something